frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/cloud-init-tab/cloud-init-tab.tsx

```diff
@@ -302,7 +307,39 @@ const CloudInitTabComponent: React.FC<ResultTabComponentProps> = ({
       )}
       <Form>
         <Title headingLevel="h5" size="lg">
-          Cloud-init
+          Cloud-init{' '}
+          <Popover
```

Comment:
can you please move this to a standalone component as this is component is quite already big?